helm/pay-wallet-values-dev.yaml

```diff
@@ -98,6 +98,9 @@ microservice-chart:
     WALLET_SEARCH_LIMIT: "20"
     SCHEDULER_REDIS_RESUME_KEYSPACE: "pay-wallet-scheduler"
     SCHEDULER_REDIS_RESUME_TTL_IN_MIN: "120"
+    SCHEDULER_REDIS_JOB_LOCK_KEYSPACE: "pay-wallet-scheduler"
+    SCHEDULER_REDIS_JOB_LOCK_TTL_SEC: "120"
+    SCHEDULER_REDIS_JOB_LOCK_WAIT_TIME_SEC: "10"
```

Comment:
this parameter should be tuned to be <= of job execution rate (as for now set to 10 seconds, but to be tuned up after soak tests). Btw i think that this timeout should be lower anyway, something like 1 sec?

Reply:
In the next PR where we integrate the lock into scheduling, we will make these parameters dependent on the job execution frequency

Reply:
Here c309e6e28bf9e9000b98b2c00d9883d983ee988b i decrease ttl and wait time. In the next PR we will do the final implementation